tools/make-v8.sh

```diff
@@ -18,8 +18,8 @@ if [[ "$ARCH" == "s390x" ]] || [[ "$ARCH" == "ppc64le" ]]; then
   export LD_LIBRARY_PATH=$BUILD_TOOLS:$LD_LIBRARY_PATH
   # Avoid linking to ccache symbolic links as ccache decides which
   # binary to run based on the name of the link (we always name them gcc/g++).
-  CC_PATH=`which -a $CC gcc | grep -v ccache | head -n 1`
-  CXX_PATH=`which -a $CXX g++ | grep -v ccache | head -n 1`
+  CC_PATH=`command -v $CC gcc | grep -v ccache | head -n 1`
+  CXX_PATH=`command -v $CXX g++ | grep -v ccache | head -n 1`
```

Comment:
This will change behaviour. `which -a` returns all matches while `command -v` returns the first match.

Reply:
Are you sure? For me `command -v node npm` returns two results, same as `which -a node npm`.

Reply:
I'll double check on Monday.

Reply:
FWIW what I'm refering to is that `which -a` returns all instances of the thing being searched for found in the directories in the `PATH` environment variable. So for example if your `gcc` is actually a symlink to `ccache`, `which -a` would return the symlink (first on the path) but also all the others and then the subsequent bits of the pipe filter out the `ccache` link.

Reply:
@richardlau thanks for pointing it out, I updated it :slightly_smiling_face:

Reply:
>> Are you sure? For me `command -v node npm` returns two results, same as `which -a node npm`.
>
> I'll double check on Monday.

@aduh95 This illustrates the difference:
```
$ which gcc
/usr/lib64/ccache/gcc
$ which -a gcc
/usr/lib64/ccache/gcc
/usr/bin/gcc
/bin/gcc
$ command -v gcc
/usr/lib64/ccache/gcc
$
```

`tools/make-v8.sh` uses `which -a` so that it can filter out the `ccache` symbolic links because linking to the symbolic link messes up `ccache`'s ability to work out if it is meant to invoke `gcc` or `g++`.